.changeset/red-hats-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+feat: upgrade vite-plugin-svelte for resolve improvements and warnings
```

Comment:
the feat: prefix confuses me a bit in a patch changeset, but making the bump a minor would be too strong as the extra warning log isn't affecting the output.

Reply:
yeah, I was debating between that and `chore:`. I'm fine with whatever folks would like to go with